xbmc/addons/AddonManager.h

```diff
@@ -561,6 +569,12 @@ namespace ADDON
 
     // Temporary path given to add-ons, whose content is deleted when Kodi is stopped
     const std::string m_tempAddonBasePath = "special://temp/addons";
+
+    /*!
+     * latest count of available updates
+     */
+    mutable std::string m_lastAvailableUpdatesCountAsString;
+    mutable std::mutex m_lastAvailableUpdatesMutex;
```

Comment:
I would name the  mutex `m_lastAvailableUpdatesCountMutex` as this is what the mutex protects.